RxCocoa/macOS/NSButton+Rx.swift

```diff
@@ -20,25 +20,31 @@ extension Reactive where Base: NSButton {
         return controlEvent
     }
 
-    /// Reactive wrapper for `state` property`.
-    public var state: ControlProperty<Int> {
-        return NSButton.rx.value(
-            base,
-            getter: { control in
-                #if swift(>=4.0)
-                    return control.state.rawValue
-                #else
+    #if swift(>=4.0)
```

Comment:
@tarunon I don't think we'll support <= 3.1. Potentially 3.2 but we'll need to evaluate what would it take to do that first.